lib/models/concerts.js

```diff
@@ -0,0 +1,40 @@
+const { ObjectID } = require("mongodb");
+const { getCollection } = require("../database");
+
+const collectionName = "concerts";
+
+async function postConcert(concert) {
+  const concertsCollection = await getCollection(collectionName);
+  const result = await concertsCollection.insertOne({ ...concert });
+
+  return result.insertedId;
+}
+
+async function getConcert(concertId) {
+  const concertsCollection = await getCollection(collectionName);
+  const objectId = new ObjectID.createFromHexString(concertId);
+  const concert = await concertsCollection.findOne({ _id: objectId });
+
+  if (!concert) {
+    throw new Error("Cannot find concerts");
```

Comment:
A more correct error message would be "Cannot find concert":

```suggestion
    throw new Error("Cannot find concert");
```